packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -211,17 +221,16 @@ export class AtomicPopover implements InitializableComponent {
     }
 
     return (
-      <div class="relative">
-        {this.renderDropdownButton()}
-        <div
-          id={this.popoverId}
-          ref={(el) => (this.popupRef = el!)}
-          part="facet"
-          class={`absolute pt-0.5 z-1 ${this.isOpen ? 'block' : 'hidden'}`}
+      <Host>
+        <atomic-focus-trap
```

Comment:
Do I need extra aria attributes here?

Reply:
no need for extra aria attributes. However, when the modal opens, the focus gets completely stuck in macOS Voiceover

Reply:
![Screen Shot 2022-08-17 at 8 31 20 AM](https://user-images.githubusercontent.com/4923043/185123046-9800877e-19ef-416a-8be1-6ddfb9a89dcb.png)

Reply:
Hmm with the VO - up/down key I cannot even access the button in the first place... But with tab it worked i'll look into it

Reply:
Got it!